From: KevinDuan <duankaiwen@kylinos.com>
Date: Wed, 5 Jun 2024 03:33:16 +0000
Subject: Translated using Weblate (Uyghur)

Currently translated at 46.2% (50 of 108 strings)

Translation: openkylin-nile-new/kylin-recorder
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-recorder/ug/
---
 translations/kylin-recorder_ug.ts | 56 +++++++++++++++++++--------------------
 1 file changed, 28 insertions(+), 28 deletions(-)

diff --git a/translations/kylin-recorder_ug.ts b/translations/kylin-recorder_ug.ts
index 2bf1877..47a25ab 100644
--- a/translations/kylin-recorder_ug.ts
+++ b/translations/kylin-recorder_ug.ts
@@ -11,7 +11,7 @@
     <message>
         <location filename="../src/clipwidget.cpp" line="197"/>
         <source>Pause</source>
-        <translation>توختاش</translation>
+        <translation>Pause</translation>
     </message>
     <message>
         <location filename="../src/clipwidget.cpp" line="112"/>
@@ -77,7 +77,7 @@
     <message>
         <location filename="../src/clipwidget.cpp" line="517"/>
         <source>OK</source>
-        <translation>ماقۇل</translation>
+        <translation>جەزملەشتۈرۈش</translation>
     </message>
     <message>
         <source>Warning</source>
@@ -110,7 +110,7 @@
     <message>
         <location filename="../src/fileitem.cpp" line="70"/>
         <source>Pause</source>
-        <translation>توختاش</translation>
+        <translation>Pause</translation>
     </message>
     <message>
         <location filename="../src/fileitem.cpp" line="75"/>
@@ -127,7 +127,7 @@
         <location filename="../src/fileitem.cpp" line="792"/>
         <location filename="../src/fileitem.cpp" line="805"/>
         <source>Warning</source>
-        <translation>ئاگاھلاندۇرۇش</translation>
+        <translation>ئاگاھلانددۇرۇش</translation>
     </message>
     <message>
         <source>Minimize</source>
@@ -240,19 +240,19 @@
     <message>
         <location filename="../src/mainwindow.cpp" line="308"/>
         <source>Minimize</source>
-        <translation>ئەڭ كىچىك ھالەت</translation>
+        <translation>كىچىكلىتىش</translation>
     </message>
     <message>
         <location filename="../src/mainwindow.cpp" line="334"/>
         <source>Close</source>
-        <translation>ياپ</translation>
+        <translation>تاقاش</translation>
     </message>
     <message>
         <location filename="../src/mainwindow.cpp" line="342"/>
         <location filename="../src/mainwindow.cpp" line="347"/>
         <location filename="../src/mainwindow.cpp" line="1752"/>
         <source>Recording</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۈن ئېلىۋاتىدۇ</translation>
     </message>
     <message>
         <location filename="../src/mainwindow.cpp" line="445"/>
@@ -276,7 +276,7 @@
     <message>
         <location filename="../src/mainwindow.cpp" line="482"/>
         <source>Right</source>
-        <translation>توغرىسى</translation>
+        <translation>ئوڭ</translation>
     </message>
     <message>
         <location filename="../src/mainwindow.cpp" line="510"/>
@@ -293,19 +293,19 @@
         <location filename="../src/mainwindow.cpp" line="1063"/>
         <location filename="../src/mainwindow.cpp" line="1116"/>
         <source>Maximize</source>
-        <translation>ئەڭ چوڭ ھالەت</translation>
+        <translation>ئەڭ چوڭ چەككە</translation>
     </message>
     <message>
         <location filename="../src/mainwindow.cpp" line="276"/>
         <source>Options</source>
-        <translation>تاللاش</translation>
+        <translation>تاللانمىلار</translation>
     </message>
     <message>
         <location filename="../src/mainwindow.cpp" line="1060"/>
         <location filename="../src/mainwindow.cpp" line="1113"/>
         <location filename="../src/mainwindow.cpp" line="1139"/>
         <source>Restore</source>
-        <translation>ئەسلىگە قايتۇرۇش</translation>
+        <translation>ئەسلىگە كەلتۈرۈش</translation>
     </message>
     <message>
         <location filename="../src/mainwindow.cpp" line="1292"/>
@@ -313,7 +313,7 @@
         <location filename="../src/mainwindow.cpp" line="1637"/>
         <location filename="../src/mainwindow.cpp" line="1837"/>
         <source>Warning</source>
-        <translation>ئاگاھلاندۇرۇش</translation>
+        <translation>ئاگاھلانددۇرۇش</translation>
     </message>
     <message>
         <location filename="../src/mainwindow.cpp" line="1292"/>
@@ -338,7 +338,7 @@
     <message>
         <location filename="../src/mainwindow.cpp" line="1394"/>
         <source>start</source>
-        <translation>باشلان</translation>
+        <translation>باشلاش</translation>
     </message>
     <message>
         <location filename="../src/mainwindow.cpp" line="1621"/>
@@ -376,7 +376,7 @@
     <message>
         <location filename="../src/miniwindow.cpp" line="92"/>
         <source>Recording</source>
-        <translation type="unfinished"></translation>
+        <translation>ئۈن ئېلىۋاتىدۇ</translation>
     </message>
     <message>
         <location filename="../src/miniwindow.cpp" line="97"/>
@@ -387,23 +387,23 @@
         <location filename="../src/miniwindow.cpp" line="104"/>
         <location filename="../src/miniwindow.cpp" line="266"/>
         <source>Pause</source>
-        <translation>توختاش</translation>
+        <translation>Pause</translation>
     </message>
     <message>
         <location filename="../src/miniwindow.cpp" line="116"/>
         <source>Restore</source>
-        <translation>ئەسلىگە قايتۇرۇش</translation>
+        <translation>ئەسلىگە كەلتۈرۈش</translation>
     </message>
     <message>
         <location filename="../src/miniwindow.cpp" line="126"/>
         <source>Close</source>
-        <translation>ياپ</translation>
+        <translation>تاقاش</translation>
     </message>
     <message>
         <location filename="../src/miniwindow.cpp" line="236"/>
         <location filename="../src/miniwindow.cpp" line="335"/>
         <source>Warning</source>
-        <translation>ئاگاھلاندۇرۇش</translation>
+        <translation>ئاگاھلانددۇرۇش</translation>
     </message>
     <message>
         <location filename="../src/miniwindow.cpp" line="236"/>
@@ -418,7 +418,7 @@
     <message>
         <location filename="../src/miniwindow.cpp" line="271"/>
         <source>Start</source>
-        <translation>باشلان</translation>
+        <translation>قوزغىتىش</translation>
     </message>
 </context>
 <context>
@@ -467,7 +467,7 @@
     <message>
         <location filename="../src/mythread.cpp" line="634"/>
         <source>Warning</source>
-        <translation>ئاگاھلاندۇرۇش</translation>
+        <translation>ئاگاھلانددۇرۇش</translation>
     </message>
     <message>
         <location filename="../src/mythread.cpp" line="634"/>
@@ -506,7 +506,7 @@
     <message>
         <location filename="../src/renamewidget.cpp" line="89"/>
         <source>Close</source>
-        <translation type="unfinished">ياپ</translation>
+        <translation>تاقاش</translation>
     </message>
     <message>
         <location filename="../src/renamewidget.cpp" line="113"/>
@@ -561,7 +561,7 @@
     <message>
         <location filename="../src/settings.cpp" line="68"/>
         <source>Close</source>
-        <translation type="unfinished">ياپ</translation>
+        <translation>تاقاش</translation>
     </message>
     <message>
         <location filename="../src/settings.cpp" line="76"/>
@@ -581,17 +581,17 @@
     <message>
         <location filename="../src/settings.cpp" line="145"/>
         <source>Format:</source>
-        <translation type="unfinished"></translation>
+        <translation>شەكلى:</translation>
     </message>
     <message>
         <location filename="../src/settings.cpp" line="159"/>
         <source>Source:</source>
-        <translation type="unfinished"></translation>
+        <translation>مەنبە :</translation>
     </message>
     <message>
         <location filename="../src/settings.cpp" line="165"/>
         <source>Microphone</source>
-        <translation type="unfinished"></translation>
+        <translation>مىكروفون</translation>
     </message>
     <message>
         <location filename="../src/settings.cpp" line="166"/>
@@ -608,7 +608,7 @@
         <location filename="../src/settings.cpp" line="297"/>
         <location filename="../src/settings.cpp" line="304"/>
         <source>Warning</source>
-        <translation>ئاگاھلاندۇرۇش</translation>
+        <translation>ئاگاھلانددۇرۇش</translation>
     </message>
     <message>
         <location filename="../src/settings.cpp" line="285"/>
@@ -641,7 +641,7 @@
         <location filename="../src/tools.cpp" line="50"/>
         <location filename="../src/tools.cpp" line="57"/>
         <source>Warning</source>
-        <translation>ئاگاھلاندۇرۇش</translation>
+        <translation>ئاگاھلانددۇرۇش</translation>
     </message>
     <message>
         <location filename="../src/tools.cpp" line="39"/>
@@ -669,7 +669,7 @@
     <message>
         <location filename="../src/menumodule.cpp" line="42"/>
         <source>Theme</source>
-        <translation>سىرتقى كۆرۈنۈش</translation>
+        <translation>Theme</translation>
     </message>
     <message>
         <location filename="../src/menumodule.cpp" line="39"/>
